static/app/views/issueDetails/groupEvents.tsx

```diff
@@ -101,17 +103,28 @@ function UpdatedSearchBar({
   }, [data]);
 
   const tagValueLoader = useCallback(
-    (key: string, search: string) => {
+    async (key: string, search: string) => {
       const orgSlug = organization.slug;
       const projectIds = [group.project.id];
 
-      return fetchTagValues({
+      const eventsDatasetValues = await fetchTagValues({
         api,
         orgSlug,
         tagKey: key,
         search,
         projectIds,
+        dataset: Dataset.ERRORS,
       });
+
+      const issuePlatformDatasetValues = await fetchTagValues({
+        api,
+        orgSlug,
+        tagKey: key,
+        search,
+        projectIds,
+        dataset: Dataset.ISSUE_PLATFORM,
+      });
+      return mergeTagValues(eventsDatasetValues, issuePlatformDatasetValues);
```

Comment:
I have updated issue details to query the Errors and IssuePlatform datasets under the assumption issue details, like issue stream, also pulls tags from events and issuePlatform, but I'm not 100% sure if that's correct. Lmk if that isn't and I can revert this change

Reply:
This is right! I can see that `monitor.slug` returns results after your change, whereas before it returned nothing. So it's working as intended!